widget/filetree.go

```diff
@@ -0,0 +1,88 @@
+package widget
+
+import (
+	"sort"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/storage"
+	"fyne.io/fyne/theme"
+)
+
+type FileTree struct {
+	Tree
+	Filter func(string) bool
+	Sorter func(string, string) bool
```

Comment:
I struggled to guess how this method would be used based on the name. Is there any way it could be clearer?
For example when this is passed to `sort.Slice` it's called `less` which helped me to guess.

It's a bit random but how about `SortAbove`?